polygon-digital-carbon/src/KlimaAggregator.ts

```diff
@@ -4,19 +4,23 @@ import { C3Retired } from '../generated/RetireC3Carbon/RetireC3Carbon'
 import { CarbonRetired, CarbonRetired1 as CarbonRetiredTokenId } from '../generated/KlimaInfinity/KlimaInfinity'
 
 import { KlimaCarbonRetirements } from '../generated/RetireC3Carbon/KlimaCarbonRetirements'
-import { Address, BigInt } from '@graphprotocol/graph-ts'
+import { BigInt, log, dataSource } from '@graphprotocol/graph-ts'
 import { loadOrCreateAccount } from './utils/Account'
 import { loadRetire } from './utils/Retire'
-import { ZERO_ADDRESS } from '../../lib/utils/Constants'
+import { AMOY_KLIMA_CARBON_RETIREMENTS_CONTRACT, ZERO_ADDRESS } from '../../lib/utils/Constants'
```

Comment:
I don't see this referenced anywhere in this file, only the subsequent one down below?

Reply:
yes, an old import; removed